README.md

```diff
@@ -23,35 +23,55 @@ Conda environment : [https://docs.conda.io/en/latest/index.html](https://docs.co
 
 ##### Create a new environment with phenomenal installed in there :
 
-    mamba create -n phm -c conda-forge -c openalea3 openalea.phenomenal
+    mamba create -n phm -c conda-forge -c openalea3 openalea.phenomenal matplotlib
     mamba activate phm
 
 ##### In an existing environment :
 
-    mamba install -c conda-forge -c openalea3 openalea.phenomenal
+    mamba install -c conda-forge -c openalea3 openalea.phenomenal matplotlib
 
 ##### (Optional) Test your installation :
 
     mamba install -c conda-forge pytest
     git clone https://github.com/openalea/phenomenal.git
     cd phenomenal/test; pytest
 
-#### From source
+### Developers
 
-    # Install dependency with conda
-    mamba env create -n phm -f conda/environment.yml 
-    mamba activate phm
-    # (on windows only)
-    mamba install vs_win-64
+#### Editable mode install
+
+    # (windows only) install VisualStudio Build Tools
 
-    # Clone phenomenal and install
+    # Clone phenomenal
     git clone https://github.com/openalea/phenomenal.git
-    cd phenomenal
-    pip install -e .
+    cd phenomenal    
+
+    # Solution 1: Build env from scratch (mostly use pip dependencies)
+    mamba env create -f conda/environment.yml
+    mamba activate phenomenal_dev
+
+    # Solution 2: use an existing env (mostly use conda dependencies)
+    mamba activate my_env
+    mamba install --only-deps -c openalea3 -c conda-forge openalea.phenomenal
+    mamba install cxx-compiler
+    pip install -e .[doc,test,plot]
 
     # (Optional) Test your installation
     cd test; pytest
 
+#### Conda package building 
+    
+    # (windows only) install VisualStudio Build Tools
+
+    # setup build env
+    mamba create -n buildenv conda-build conda-forge-pinning
+    mamba activate buildenv
+
+    # build package (see openalea guidelines for other options)
+    conda build . -m $CONDA_PREFIX/conda_build_config.yaml --python "3.12.* *_cpython"
+    # (on windows):
+    conda build .  -m %CONDA_PREFIX%\conda_build_config.yaml --python "3.12.* *_cpython"
+
```

Comment:
Not sure why users have to build their own conda package.
For me, this is a wrong practice, with lots of side effects.

Is it to build with a compiler phenomenal? To discuss where we can explain this globally to openalea

Reply:
To me, it is mostly to debug locally a build failling on CI.
But surely not for deploying the module !

I open a PR on openalea.rtfd.io to explain better what to do at oa level

Reply:
Okay I remove the section from readme and keep it in guidelines only